### DISCUSSION TITLE
Save Data, Reset Pi and import data

### DISCUSSION THREAD

**Discussion by chrisgrnwld** [teslamate-org/teslamate#1296] [Q&A]
Hi,
I think, I messed up my raspberry pi. My idea is to reset the pi, install teslamate and import the old data. How can I do that? Does it work by simply copying the data to the right folders? If so: where can I find this data?
Sorry for the stupid questions but I‘m more then new to this
Cheers 
Chris

**Comment by chrisgrnwld** [comment]
Edit: I tried to do the backup as shown in the documentation, but it keeps saying that it can‘t find the folder. Also if I search for the db I can‘t find anything...
But the data must be somewhere because I can still see it in Grafana Dashboard...

**Comment by DrMichael** [comment]
Which old data do you want to import?

If you have a backup of the database, then a fresh installation would be the best bet. But first remove any images and volumes from docker.
